test/unit/calculators/holiday_entitlement_test.rb

```diff
@@ -273,5 +273,24 @@ class HolidayEntitlementTest < ActiveSupport::TestCase
         end
       end
     end
+
+    context "calculate entitltment on hours worked per week" do
```

Comment:
Spotted a small typo here - should be 'entitlement'

Reply:
```suggestion
    context "calculate entitlement on hours worked per week" do
```